core/apps.py

```diff
@@ -1,12 +1,37 @@
 from django.apps import AppConfig
 from django.conf import settings as s
 
+import os
+import urllib
+
 class CoreConfig(AppConfig):
     name = 'core'
     verbose_name = 'Open ONI Core'
 
     def ready(self):
         # Define internal settings derived from local settings
+            # BASE_URL-dependent settings
+        url = urllib.parse.urlparse(s.BASE_URL)
+
+        s.ALLOWED_HOSTS = [url.netloc]
```

Comment:
Let's check this for existing data - some people may want multiple hosts, especially for development (I sometimes toss in "*")

Reply:
I could have it add in the wildcard for when DEBUG is True :thinking:

Reply:
I think a check for a setting is a safer bet.  We can't know what people will want to allow.

Reply:
If we don't want to allow this, we need to document it *very* very clearly.  It's a core Django setting that's well-documented on their site.

Reply:
Okay, I'll just move this back out into `local_settings_example.py` so it will be easy for people to define as they like alongside other general settings.